mail-client/alot/alot-0.6.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1 vcs-snapshot
+
+DESCRIPTION="Experimental terminal UI for net-mail/notmuch written in Python"
+HOMEPAGE="https://github.com/pazz/alot"
+SRC_URI="${HOMEPAGE}/tarball/${PV} -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc test"
+
+DEPEND="
+	doc? ( dev-python/sphinx[${PYTHON_USEDEP}] )
+	test? ( dev-python/mock[${PYTHON_USEDEP}] )
+	"
+RDEPEND="
+	>=app-crypt/gpgme-1.8.0-r3[python,${PYTHON_USEDEP}]
+	>=dev-python/configobj-4.7.0[${PYTHON_USEDEP}]
+	dev-python/python-magic[${PYTHON_USEDEP}]
+	>=dev-python/twisted-core-10.2.0[${PYTHON_USEDEP}]
+	>=dev-python/urwid-1.1.0[${PYTHON_USEDEP}]
+	>=dev-python/urwidtrees-1.0[${PYTHON_USEDEP}]
+	net-mail/mailbase
+	>=net-mail/notmuch-0.13[crypt,python,${PYTHON_USEDEP}]
+	"
+
+ALOT_UPDATE=""
+
+pkg_setup() {
+	if has_version "<${CATEGORY}/${PN}-0.3.2"; then
+		ALOT_UPDATE="yes"
+	fi
+}
+
+src_prepare() {
+	find "${S}" -name '*.py' -print0 | xargs -0 -- sed \
+		-e '1i# -*- coding: utf-8 -*-' -i || die
+
+	distutils-r1_src_prepare
+
+	local md
+	for md in *.md; do
+		mv "${md}" "${md%.md}"
```

Comment:
is this worth it really? Instead of just defining `DOCS=( blabla.md )` in global scope

Reply:
Maybe not it comes from an old version, I switched it to DOCS statement